config.py

```diff
@@ -192,9 +191,14 @@ def init_app(cls, app):
                    getLogger('elasticsearch'),
                    getLogger('redislite')]
 
+        # Sadly, this does not work:
```

Comment:
because this line https://github.com/pallets/werkzeug/blob/368ccb07d097660156eb19a0ec2042bc53e68238/werkzeug/_internal.py#L83 you cannot change it this way. but keep it like this